### PR TITLE
Code gen

### PR DESCRIPTION
**Pull request by FrankBakkerNl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "@docusaurus/preset-classic": "^2.0.0-beta.9",
     "classnames": "^2.2.6",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "yarn": "^1.22.17"
```
Comment (ludeeus) [inline package.json:21]:
```suggestion
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "yarn": "^1.22.17"
```

File: docs/api/hass_model/hass_model_codegen.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+NetDeamon has a generator that creates code based on the Entities and services in Home Assitant. This makes it possible to navigate all Entities and Services using intellisense.
+
+The code generator is installed as a global dotNet tool via the following command:
+
+```
+dotnet tool install -g JoySoftware.NetDaemon.HassModel.CodeGen --version 21.46.0-beta
+```
+
+*While in preview, it is required to include the -version of the most recent version explcitily.*
```
Comment (helto4real) [inline docs/api/hass_model/hass_model_codegen.md:14]: Maybe include “please check nuget what latest version is” so we don’t have to maintain this.

File: docs/api/hass_model/hass_model_generated_entities.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+LightEntity livingRoomLight = _myEntities.Light.LivingRoomLight;
+```
+
+`Entities` is a generated class that provides acces to all your entities in Home Assitant. The object returnd by `_myEntities.Light.LivingRoomLight` is of type `LightEntity` which is also generated. This specific instance represents the `light.living_room_light` entity in Home Assitant.
+
+The LightEntity class provides strong typed access to the state and attributes of this entity like 
+
+```csharp
+if (livingRoomLight.State == "on" && livingRoomLight.Attributes.Brightness > 100) // ...
```
Comment (helto4real) [inline docs/api/hass_model/hass_model_generated_entities.md:21]: We could also show alternative ‘IsOn()’

**Review by helto4real — APPROVED** [review]
Just a few suggestions, overall looks good.

**Pull request merged**